packages/telemetry/README.md

```diff
@@ -0,0 +1,278 @@
+# @packages/telemetry
+
+This package is convenience wrapper built around open telemetry to allow us to gain insights around how cypress is used and help us prevent performance regressions.
+
+## tl;dr
+
+Telemetry in cypress is off by default. To enable telemetry in cypress set `CYPRESS_INTERNAL_ENABLE_TELEMETRY="true"`.
+
+Telemetry data is sent to the cloud `/telemetry` endpoint.
+
+For the **cypress cloud project only** we forward the telemetry data to honeycomb. For all other projects telemetry data is not stored.
+
+Staging: https://ui.honeycomb.io/cypress/environments/cypress-app-staging/datasets/cypress-app/home
+
+Prod: https://ui.honeycomb.io/cypress/environments/cypress-app/datasets/cypress-app/home
+
+## Design
+
+At a very high level we use open telemetry to collect data and it to honeycomb. There are three different processes that collect data, the server, the child process and the browser. The child process and browser forward collected spans to the server where the telemetry data is encrypted and sent to an entrypoint on the Cypress server. The Cypress server then decrypts the telemetry data and decides what to do with it. Today, if the attached project is the Cypress project we forward the data to honeycomb, all other projects are ignored.
```

Comment:
```suggestion
At a very high level we use open telemetry to collect data and send it to honeycomb. There are three different processes that collect data, the server, the child process and the browser. The child process and browser forward collected spans to the server where the telemetry data is encrypted and sent to the Cypress Cloud. The Cypress Cloud then decrypts the telemetry data and decides what to do with it. Today, if the attached project is the Cypress project we forward the data to honeycomb, all other projects are ignored.
```

Reply:
should there be a colon after `collect data`?